base/system/sys_info.cc

```diff
@@ -25,8 +25,15 @@ namespace {
 // For M99, 45% of devices have 2GB of RAM, and 55% have more.
 constexpr uint64_t kLowMemoryDeviceThresholdMB = 1024;
 #else
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+#if !BUILDFLAG(IS_ANDROID)
+// Updated Desktop default threshold to match the Android 2021 definition.
+constexpr uint64_t kLowMemoryDeviceThresholdMB = 2048;
```

Comment:
Why is this necessary?

Reply:
This is to remove the warning of the unused kLowMemoryDeviceThresholdMB.

Reply:
Okay, then this can be simplified—it looks right now like when it's declared and when it's used don't match (i.e. it's declared with SbMedia on linux but not used), so I would suggest changing this block from

```
#if BUILDFLAG(USE_STARBOARD_MEDIA)
#if !BUILDFLAG(IS_ANDROID)
// Updated Desktop default threshold to match the Android 2021 definition.
constexpr uint64_t kLowMemoryDeviceThresholdMB = 2048;
#endif //!BUILDFLAG(IS_ANDROID)
#else //BUILDFLAG(USE_STARBOARD_MEDIA)
// Updated Desktop default threshold to match the Android 2021 definition.
constexpr uint64_t kLowMemoryDeviceThresholdMB = 2048;
#endif //BUILDFLAG(USE_STARBOARD_MEDIA)
```

to

```
#if !BUILDFLAG(USE_STARBOARD_MEDIA)
// Updated Desktop default threshold to match the Android 2021 definition.
constexpr uint64_t kLowMemoryDeviceThresholdMB = 2048;
#endif //!BUILDFLAG(USE_STARBOARD_MEDIA)
```

or if the android buildflag is still needed, you can add that back in, but you shouldn't need an if/else macro